client/my-sites/sidebar-unified/static-data/jetpack-fallback-menu.js

```diff
@@ -0,0 +1,248 @@
+		},
+		{
+			icon: JETPACK_ICON,
+			slug: 'jetpack',
+			title: translate( 'Jetpack' ),
+			type: 'menu-item',
+			url: `/activity-log/${ siteDomain }`,
+			children: [
+				{
+					parent: 'jetpack',
+					slug: 'jetpack-activity-log',
+					title: translate( 'Activity Log' ),
+					type: 'submenu-item',
+					url: `/activity-log/${ siteDomain }`,
+				},
+				{
+					parent: 'jetpack',
+					slug: 'jetpack-backup',
+					title: translate( 'Backup' ),
+					type: 'submenu-item',
+					url: `/backup/${ siteDomain }`,
+				},
+				{
+					parent: 'jetpack',
+					slug: 'jetpack-scan',
+					title: translate( 'Scan' ),
+					type: 'submenu-item',
+					url: `/scan/${ siteDomain }`,
+				},
+				{
+					parent: 'jetpack',
+					slug: 'jetpack-search',
+					title: translate( 'Search' ),
+					type: 'submenu-item',
+					url: `/jetpack-search/${ siteDomain }`,
+				},
+			],
+		},
+		{
+			type: 'separator',
+		},
+		{
+			icon: 'dashicons-admin-appearance',
+			slug: 'themes-php',
+			title: translate( 'Appearance' ),
+			type: 'menu-item',
+			url: `/themes/${ siteDomain }`,
+			children: [
+				{
+					parent: 'themes.php',
+					slug: 'themes-php',
+					title: translate( 'Themes' ),
+					type: 'submenu-item',
+					url: `/themes/${ siteDomain }`,
+				},
+				{
+					parent: 'themes.php',
+					slug: 'themes-customize',
+					title: translate( 'Customize' ),
+					type: 'submenu-item',
+					url: `https://${ siteDomain }/wp-admin/customize.php`,
+				},
+			],
+		},
+		{
+			icon: 'dashicons-admin-plugins',
+			slug: 'plugins',
+			title: translate( 'Plugins' ),
+			type: 'menu-item',
+			url: `/plugins/${ siteDomain }`,
+		},
+		{
+			icon: 'dashicons-admin-users',
+			slug: 'users-php',
+			title: translate( 'Users' ),
+			type: 'menu-item',
+			url: `/people/team/${ siteDomain }`,
+		},
+		{
+			icon: 'dashicons-admin-tools',
+			slug: 'tools-php',
+			title: translate( 'Tools' ),
+			type: 'menu-item',
+			url: `/marketing/tools/${ siteDomain }`,
+			children: [
+				{
+					parent: 'tools.php',
+					slug: 'tools-marketing',
+					title: translate( 'Marketing' ),
+					type: 'menu-item',
+					url: `/marketing/tools/${ siteDomain }`,
+				},
+				{
+					parent: 'tools.php',
+					slug: 'tools-earn',
+					title: translate( 'Earn' ),
+					type: 'menu-item',
+					url: `/earn/${ siteDomain }`,
+				},
+				{
+					parent: 'tools.php',
+					slug: 'tools-import',
+					title: translate( 'Import' ),
+					type: 'submenu-item',
+					url: `https://${ siteDomain }/wp-admin/import.php?calypsoify=0`,
+				},
+				{
+					parent: 'tools.php',
+					slug: 'tools-export',
+					title: translate( 'Export' ),
+					type: 'submenu-item',
+					url: `https://${ siteDomain }/wp-admin/import.php?calypsoify=0`,
+				},
+			],
+		},
+		{
+			icon: 'dashicons-admin-settings',
+			slug: 'options-general-php',
+			title: translate( 'Settings' ),
+			type: 'menu-item',
+			url: `/settings/general/${ siteDomain }`,
+		},
+		{
+			icon: WPADMIN_ICON,
```

Comment:
There is a dashicon that we can use here: https://developer.wordpress.org/resource/dashicons/#wordpress-alt

```suggestion
			icon: 'dashicons-wordpress-alt',
```

It's what I used in https://github.com/Automattic/jetpack/pull/18586: https://github.com/Automattic/jetpack/blob/ce0d198cc3ce6587307dac8b387c83a3838b2e6a/projects/plugins/jetpack/modules/masterbar/admin-menu/class-jetpack-admin-menu.php#L77

Reply:
Ohhh, thanks for that fe6d2d8